fili-core/src/test/groovy/com/yahoo/bard/webservice/table/availability/PermissiveAvailabilitySpec.groovy

```diff
@@ -0,0 +1,80 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.table.availability
+
+import com.yahoo.bard.webservice.data.config.names.TableName
+import com.yahoo.bard.webservice.metadata.TestDataSourceMetadataService
+import com.yahoo.bard.webservice.table.Column
+import com.yahoo.bard.webservice.table.resolver.DataSourceConstraint
+import com.yahoo.bard.webservice.util.SimplifiedIntervalList
+import org.joda.time.Interval
+
+import spock.lang.Specification
+
+class PermissiveAvailabilitySpec extends Specification  {
+
+    PermissiveAvailability permissiveAvailability
+    Column column1
+    Column column2
+    Interval interval1
+    Interval interval2
+
+    def setup() {
+
+        column1 = new Column('column1')
+        column2 = new Column('column2')
+
+        interval1 = new Interval('2010-01-01/2020-12-31')
+        interval2 = new Interval('2017-01-01/2020-12-31')
+
+        permissiveAvailability = new PermissiveAvailability(
+                TableName.of('table'),
+                [column1, column2].toSet(),
```

Comment:
In Groovy, it's generally more idiomatic to use the coercion operator: `[col1, col2] as Set`